module.exports.unexpectedErrorWhileCommandExecution = '명령어 실행 중에 알 수 없는 오류가 발생하여 네트워크에 영향을 미칠 가능성을 확인 중이라고 미사카는 보고합니다.'
